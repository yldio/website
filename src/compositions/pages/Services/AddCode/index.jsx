import React from 'react';
import { H2, Copy } from 'components/Typography';
import Container from 'components/Container';
import styled from 'styled-components';
import breakpoints from 'styles/breakpoints';
import remcalc from 'remcalc';
import Card from 'components/Card';
import add from '../assets/remote-header.png';

export const EngHeader = styled.section`
    background: url('${add}');
    background-position: left top;
    background-size: cover;
    height: ${remcalc(460)};
    padding-top: ${remcalc(300)};
    margin-bottom: ${remcalc(80)};
`;

export const Row = styled.div`
  display: flex;
  margin: ${remcalc(80)} 0;
  justify-content: space-between;
  flex-wrap: wrap;

  & > * {
    width: 100%;
    flex-shrink: 0;

    ${breakpoints.tablet`
        width: 45%;
  `};
  }
`;

const Wrapper = styled.div`
  display: flex;
  justify-content: center;

  ${breakpoints.tablet`
    justify-content: flex-end;
    margin-top: ${remcalc(-155)};
  `};
`;

const AddCode = () => (
  <section data-selector="services:add-code">
    <EngHeader>
      <Container>
        <H2 white slim>
          Remote Dedicated Services - Addcode
        </H2>
      </Container>
    </EngHeader>
    <Container>
      <Row>
        <Copy>
          If you are looking for iterative and Agile development strategies;
          cooperative development procedures; fast and easy communication; rapid
          prototyping; and user-centric design, remote teams nearshore are is
          the option for you.
          <br />
          <br />
          Utilizing our Addcode, our remote option brings benefits. Meaning this
          method provides advantages in both the initial cost standpoint, and in
          its risk-averseness.
        </Copy>
        <Wrapper>
          <Card
            title="ADDCODE BRINGS YOU"
            points={[
              'Scalability',
              'Language skills and proficiency',
              'Tech Skills',
              '	Types of services',
              'Complexity of work',
              'Geography - same time zone',
              'Talent offering',
              'Choice of location - Porto & Lisbon, Manchester ',
            ]}
          />
        </Wrapper>
      </Row>
    </Container>
  </section>
);

export default AddCode;
