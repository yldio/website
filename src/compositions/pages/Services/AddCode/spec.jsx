import React from 'react';
import { shallow } from 'enzyme';
import AddCode from './index';

test('compositions/pages/Services/AddCode snapshot', () => {
  const wrapper = shallow(<AddCode />);

  expect(wrapper).toMatchSnapshot();
});
