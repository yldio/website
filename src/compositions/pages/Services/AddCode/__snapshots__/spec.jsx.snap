// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services/AddCode snapshot 1`] = `
<section
  data-selector="services:add-code"
>
  <AddCode__EngHeader>
    <Container>
      <Typography__H2
        slim={true}
        white={true}
      >
        Remote Dedicated Services - Addcode
      </Typography__H2>
    </Container>
  </AddCode__EngHeader>
  <Container>
    <AddCode__Row>
      <Typography__base>
        If you are looking for iterative and Agile development strategies; cooperative development procedures; fast and easy communication; rapid prototyping; and user-centric design, remote teams nearshore are is the option for you.
        <br />
        <br />
        Utilizing our Addcode, our remote option brings benefits. Meaning this method provides advantages in both the initial cost standpoint, and in its risk-averseness.
      </Typography__base>
      <AddCode__Wrapper>
        <Card
          points={
            Array [
              "Scalability",
              "Language skills and proficiency",
              "Tech Skills",
              "	Types of services",
              "Complexity of work",
              "Geography - same time zone",
              "Talent offering",
              "Choice of location - Porto & Lisbon, Manchester ",
            ]
          }
          title="ADDCODE BRINGS YOU"
        />
      </AddCode__Wrapper>
    </AddCode__Row>
  </Container>
</section>
`;
